CODEOWNERS

```diff
@@ -14,3 +14,5 @@
 /*/extensions/transport_sockets/alts @htuch @yangminzhu
 # sni_cluster extension
 /*/extensions/filters/network/sni_cluster @rshriram @lizan
+#  extension
+/*/extensions/filters/network/forward_original_sni @lizan
```

Comment:
name yourself?

Reply:
@lizan Sure, let me extract the non-extension part into a separate PR, I am in favor of separate PRs.